keyboards/keebio/bdn9/keymaps/vial/rules.mk

```diff
@@ -0,0 +1,11 @@
+LTO_ENABLE = yes
+COMMAND_ENABLE = no
+ifdef RGB_MATRIX_ENABLE
```

Comment:
The build fails on the BDN9 rev1 due to firmware size so I used that as a check to enable only for the rev2. If there's a way to check for which rev is being built that would be more correct but I'm not sure how to do that.

Reply:
If that's the case, can you put the keymap under `keebio/bdn9/rev2/keymaps/vial`? Otherwise, the build bot isn't even currently trying to build rev2 so it's kinda pointless to have vial support for it in vial-qmk.